templates/docs/component-status.md

```diff
@@ -22,18 +22,12 @@ When we add, make significant updates, or deprecate a component we update their
     </tr>
   </thead>
   <tbody>
-    <!-- 2.18 -->
-    <tr>
-      <th><a href="/docs/patterns/breadcrumbs">Breadcrumbs</a></th>
-      <td><div class="p-label--updated">Updated</div></td>
-      <td>2.18.0</td>
-      <td>We updated the markup of the breadcrumbs component to comply with accessibility requirements. The class <code>.p-breadcrumbs</code> is now moved onto a <code>&lt;nav&gt;</code> element, the unordered list has been changed to an ordered one that has a class <code>.p-breadcrumbs__items</code>. </td>
-    </tr>
+    <!-- 2.19 -->
     <tr>
-      <th><a href="/docs/patterns/breadcrumbs#deprecated-markup">Breadcrumbs</a></th>
-      <td><div class="p-label--deprecated">Deprecated</div></td>
-      <td>2.18.0</td>
-      <td>Top level <code>&lt;ul&gt;</code> with a class <code>.p-breadcrumbs</code> is now deprecated for the Breadcrumbs pattern.</td>
+      <th><a href="/docs/base/tables#icons">Table icons</a></th>
+      <td><div class="p-label--new">New</div></td>
+      <td>2.19.0</td>
+      <td>We added a <code>p-table__cell--with-icon</code> class to properly align icons in table cells.</td>
```

Comment:
Is --with-icon an appropriate name when the cell doesn't have an icon, but simply aligns with other cells that do?
Would an alternative class name like --icon-placeholder fit better?

Reply:
@lyubomir-popov You mean to replace `--with-icon` with `--icon-placeholder` or have both (that will technically do the same)?

Reply:
Yes, replace